transit/world_feed/world_feed.cpp

```diff
@@ -1068,14 +1068,23 @@ void WorldFeed::FillTransfers()
 
     TransferData data;
     data.m_stopsIds = {stop1Id, stop2Id};
-    data.m_point = m_stops.m_data.at(stop1Id).m_point;  // TODO maybe change?
 
-    std::tie(std::ignore, inserted) = m_transfers.m_data.emplace(transitId, data);
-    if (inserted)
+    auto & stop1 = m_stops.m_data.at(stop1Id);
+    auto & stop2 = m_stops.m_data.at(stop2Id);
+
+    // Coordinate of the transfer is the midpoint between two stops.
+    data.m_point = stop1.m_point.Mid(stop2.m_point);
+
+    if (m_transfers.m_data.emplace(transitId, data).second)
     {
       EdgeTransferId const transferId(stop1Id, stop2Id);
+
       std::tie(std::ignore, inserted) =
           m_edgesTransfers.m_data.emplace(transferId, transfer.min_transfer_time);
+
+      LinkTransferIdToStop(stop1, transitId);
+      LinkTransferIdToStop(stop1, transitId);
```

Comment:
Вроде бы здесь надо к первой и ко второй остановке привязывать пересадку.

```suggestion
      LinkTransferIdToStop(stop1, transitId);
      LinkTransferIdToStop(stop2, transitId);
```